### ISSUE TITLE
accessible name and visible label of new chat button do not match

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3817]
**Issue Description:** The visible label for the button is "New Chat" which is a good description for the purpose of the button. However the current accessible and programatic name name of the button for assistive technology is "nav-new-chat-btn" from the aria-label attribute. The accessible name should match the visible label. 

Related to issue #3819 

<img width="1684" alt="Screenshot 2024-08-28 at 9 58 17 AM" src="https://github.com/user-attachments/assets/6d22e967-7ae1-431b-b12a-ead7d1413c69">

**WCAG Criteria**: 2.5.3 Label in Name - A 

**Screen shot**: [https://share.zight.com/p9uojPNK](https://share.zight.com/p9uojPNK)

**Snippet**: `<button id="nav-new-chat-btn" aria-label="nav-new-chat-btn" class="text-token-text-primary" data-state="closed">`

**Notes**: The name of the button should be "New Chat".  Names should be descriptive of the purpose of the button but details like that it is a button, or that it is in the navigation region (NAV-new-chat-BTN) are redundant since they will be described by the screen reader as the landmark and role. When creating names it is best to use human readable, full text so that the name is pronounced correctly aloud.
